client/src/Signup/Signup.js

```diff
@@ -230,7 +227,7 @@ export function Signup() {
               setUser({ ...user, language: event.target.value })
```

Comment:
**suggestion (non-blocking)**: What do you think of setting the initial value of `language` [here](https://github.com/pieforproviders/pieforproviders/pull/193/files#diff-1f11d77c4fc8b72a41680a8f3e750206R23) to `i18n.language`?